src/main/java/modtrekt/logic/parser/module/ModuleParser.java

```diff
@@ -34,13 +39,23 @@ private static Module sendGet(String code) throws IOException, InterruptedExcept
 
         HttpResponse<String> response = httpClient.send(request, HttpResponse.BodyHandlers.ofString());
```

Comment:
The fallback does not work when the connection cannot be established (e.g. WiFi disabled) because a `java.net.ConnectionException` at this line, so there won't be a response to get the status code for.

`java.net.ConnectionException` is a subclass of `IOException` which means it gets caught by `AddModuleCommand`, which means this edge case was taken care of unintentionally.

Consequently, it'll instead throw a `CommandException` asking the user to use the verbose command - which I assume is unintended behavior as it should retrieve the module from the file.

Reply:
Good find! Will fix that